README.md

```diff
@@ -107,13 +107,34 @@ explore a commercial market, please contact us for a license agreement.
 
 
 
-## Development
+# Development
 For code formatting, [`black`](https://github.com/psf/black) and [`isort`](https://github.com/PyCQA/isort) are used.
 Type hints should be added as much as possible.
 Types are checked using [`mypy`](https://github.com/python/mypy).
 
 Before code is pushed, make sure that the `make lint` command succeeds, which runs `black`, `isort`, `flake8` and `mypy`.
 
+## Branches and releases
+A form of "git flow" is used for branch and release management. The main active branch is `develop`.
+New features are developed in new separate branches, preferrably with a name representing the new feature.
+To get the new features in the main branch, open a pull request for merging the feature branch into the `develop` branch.
+A `master` or `main` branch is not used.
+
+When a release is made, a new branch `release-X.Y` (e.g. `release-0.12`) is created from the `develop` branch.
+Only small fixes (patches) may be pushed onto this release branch. Bigger new features need to go into separate branches, merged into `develop`, and will hence end up in a later release.
+**Tags** are *only* applied on commits in the release branch. The first tag on a new release branch needs to be `vX.Y.0`, e.g. `v0.12.0`.
+Patches (i.e. commits on the release branch) may then be tagged with `vX.Y.1`, `vX.Y.2` etc.
```

Comment:
When are patches expected to be merged into `develop`?

Reply:
Added info on this

Reply:
Got it, looks good.